guides/v2.2/cloud/cdn/fastly-waf-service.md

```diff
@@ -1,47 +1,45 @@
-- **WAF onboarding and enablement**—Magento works with you to plan and schedule the WAF rollout to deploy the WAF, tune the default WAF policy in your environment, and enable the service.
-- **Operations and maintenance support**—
-  - Magento and Fastly set up and manage your WAF logs and alerts.
-  - Customer support tickets related to WAF issues that block legitimate traffic are triaged as Priority 1 issues.
-  - WAF version upgrades to ensure immediate coverage for new or evolving exploits. See [WAF maintenance and upgrades](#waf-maintenance-and-updates).
+-  **PCI compliance**—WAF enablement ensures that {{ site.data.var.ee }} storefronts in production environments meet PCI DSS 6.6 security requirements.
+-  **Default WAF policy**—The default WAF policy, configured and maintained by Fastly, provides a collection of security rules tailored to protect your {{ site.data.var.ee }} web applications from a wide range of attacks, including injection attacks, malicious inputs, cross-site scripting, data exfiltration, HTTP protocol violations, and other [OWASP Top Ten](https://www.owasp.org/index.php/Top_Ten) security threats.
+-  **WAF onboarding and enablement**—Magento deploys and enables the default WAF policy in your production environment within 2-3 weeks of the environment being fully provisioned.
+-  **Operations and maintenance support**—
+   -  Magento and Fastly set up and manage your WAF logs and alerts.
+   -  Magento triages customer support tickets related to WAF issues that block legitimate traffic as Priority 1 issues.
+   -  Automated WAF version upgrades ensure immediate coverage for new or evolving exploits. See [WAF maintenance and upgrades](#waf-maintenance-and-updates).
 
-Both the {{ site.data.var.ece }} Pro and Starter subscriptions include the Managed Cloud WAF at no additional cost. The WAF service is available only on Production environments.
+The WAF service is available on Production environments for all {{ site.data.var.ece }} Pro and Starter plans at no additional costs.
 
 {:.bs-callout .bs-callout-tip}
 For additional information about maintaining PCI compliance for your {{ site.var.data.ece }} stores, see [Magento Approach to PCI compliance](https://magento.com/pci-compliance).
 
 ### Enabling the WAF
 
-Fastly implements the managed web application firewall for {{ site.data.var.ece }} using the Fastly CDN service. You do not have to install or maintain any hardware or software.
+Magento enables the WAF service on new accounts within 2-3 weeks after final provisioning. The WAF is implemented through Fastly CDN service, you do not have to install or maintain any hardware or software.
```

Comment:
When it says "final" provisioning, are there other "provisionings" they will experience? This is worded differently than up above, so makes me think there is a slight difference. It says "within 2-3 weeks of the environment being fully provisioned." I think the one here is more elegant than the earlier one.

```suggestion
Magento enables the WAF service on new accounts within 2 to 3 weeks after provisioning is final. The WAF is implemented through Fastly CDN service; you do not have to install or maintain any hardware or software.
```